packages/kbn-i18n/src/index.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import './angular_i18n';
+
+export * from './react_i18n';
+
+export { default as I18n } from './core';
+export { default as i18n } from './i18n';
```

Comment:
This file has some issues.
* I don't see the point of the file `src/i18n.js` as it currently stands. I agree with Aleh about renaming `src/core.js` to `src/i18n.js`, and then we can export the `I18n` class.
* what is the point of the unused import on line 20?
* looking at how the docs read, people are only going to use the `I18n` class? What is the point of the `i18n` class? is it just to provide a default instance of the class? Is that used anywhere in this library, or could it just be created by consumers of the library?

Reply:
Ah, I see the `i18n` instance is what's used to create the angular and react providers. Let me give my last point (above) a little more thought.